dashboard/app/models/school.rb

```diff
@@ -282,29 +287,173 @@ def self.seed_from_s3
           }
         end
       end
+
+      CDO.log.info "Seeding 2018-2019 public and charter school data."
+      # Download link found here: https://nces.ed.gov/ccd/files.asp#Fiscal:2,LevelId:7,SchoolYearId:33,Page:1
+      # Actual download link: https://nces.ed.gov/ccd/data/zip/ccd_sch_029_1819_w_1a_091019.zip
+      AWS::S3.seed_from_file('cdo-nces', "2018-2019/ccd/ccd_sch_029_1819_w_1a_091019.csv") do |filename|
+        merge_from_csv(filename, {headers: true, encoding: 'ISO-8859-1:UTF-8', quote_char: "\x00"}, true, is_dry_run: false, new_attributes: ['last_known_school_year_open', 'school_category']) do |row|
+          {
+            id:                           row['NCESSCH'].to_i.to_s,
+            name:                         row['SCH_NAME'].upcase,
+            # Four schools with addresses longer than 50 characters (DB column limit)
+            # Also four schools with second address line longer than 30 characters.
```

Comment:
What happens with those schools?

Reply:
I'm truncating them (was getting DB errors on import without the truncation, figured cutting these four schools down in length wasn't a huge deal and a lot easier than expanding the allowed character limit for the columns via migration).